test/JSTest.Runner.UnitTests/TestRunner/TestFrameworks/Jest/JestReporterTests.ts

```diff
@@ -0,0 +1,162 @@
+import { JestCallbacks } from '../../../../../src/JSTest.Runner/TestRunner/TestFrameworks/Jest/JestCallbacks';
+import { TestUtils } from '../../../TestUtils';
+import { } from '../../../../../src/JSTest.Runner/TestRunner/TestFrameworks/Jest/JestReporter';
+import * as Assert from 'assert';
+import { TestOutcome } from '../../../../../src/JSTest.Runner/ObjectModel/Common';
+
+describe('JestReporter suite', () => {
+    const logger = new TestUtils.MockDebugLogger();
+    // tslint:disable-next-line:no-require-imports
+    const jestReporter = require('../../../../../src/JSTest.Runner/TestRunner/TestFrameworks/Jest/JestReporter');
+
+    it('onRunComplete will call handleSessionDone', (done) => {
+        jestReporter.INITIALIZE_REPORTER({
+            handleSessionDone: () => {
+                done();
+            }
+        });
+        
+        const reporter = new jestReporter();
+        reporter.onRunComplete();
+    });
+
+    it('JestReporter will handle error message', (done) => {
+        jestReporter.INITIALIZE_REPORTER({
+            handleErrorMessage: (msg) => {
+                Assert.equal(msg, 'failure');

```

Comment:
>Assert.equal(msg, 'failure'); [](start = 16, length = 29)

will the test fail.. if this call isn't received..?

Reply:
Yes, the method `done()` needs to be called for the test to be marked complete otherwise the test times out